src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setEx] = useState(null);
+  const removeEx = (workout) => {
+    setEx(workout);
+    setTotal((count) => count - 1);
+
+    const exerciseIndex = selectedExercises.findIndex(
+      (i) => i.name === workout.name
+    );
+    selectedExercises.splice(exerciseIndex, 1);
+
+    console.log("Exercise: " + workout.name);
+    console.log("Index: " + exerciseIndex);
+  };
+
+  const toggleDrawer = (anchor, open) => (event) => {
+    if (
+      event.type === "keydown" &&
+      (event.key === "Tab" || event.key === "Shift")
+    ) {
+      return;
+    }
+
+    setDrawer({ ...drawer, [anchor]: open });
+  };
+
+  const [drawer, setDrawer] = React.useState({
+    top: false,
+    left: false,
+    bottom: false,
+    right: false,
+  });
+
+  const [openSnackbar, setSnackbar] = React.useState(false);
+  const [snackbarName, setSnackbarName] = React.useState("");
+
+  const handleCloseSnackbar = (event, reason) => {
```

Comment:
What is reason?? I don't see reason being passed in to this function anywhere.

Reply:
This might be another Material UI documentation thing. We might be able to remove the "if statement" and the "reason variable" as well. From my understanding, it might be a placeholder for having a "clickaway" functionality. But again, we probably don't need it if you think we could be fine without it.